src/main/java/Main.java

```diff
@@ -1,5 +1,5 @@
-public class Main {
-    public static void main(String[] args) {
-        // your code here
-    }
-}
+public class Main {
+	public static void main(String []args)  {

```

Comment:
Usually it's `String[] args` in the parameter list with (note the spacing).

Reply:
fixed